department-of-veterans-affairs/caseflow
# Show cancelled appeals in search and list all cases

Resolves #14221 

### Description
Removed the filter on Appeals Finder that filtered out Cancelled appeals. 

Search results for finding appeals via File Number will now show cancelled appeals. 
View All Cases on the case details page will also show cancelled appeals.

### Acceptance Criteria
- [x] Cancelled appeals always show in search results
  - [x] Cancelled appeals appear in search results when searching via the Veteran ID 
  - [x] Cancelled appeals appear in search results when searching via the docket ID

### Testing Plan
Search
1. Sign in as a judge or attorney & find an active appeal in your Queue
2. Note the Veteran ID
3. Correct Issues and mark the issues as Removed
4. Go to search and search for the veteran ID, confirm that appeal is returned

View All Cases
1. Sign in as a Judge and find an active appeal with multiple cases in 'View All Cases'
2. Cancel the appeal via removal of the issues
3. Confirm on 'View All Cases' the cancelled appeal is shown

### User Facing Changes

Page | BEFORE|AFTER
---|---|---
Case Details View All | ![image](https://user-images.githubusercontent.com/6129479/87164563-e9b41e80-c296-11ea-9fab-79eeb0532dbd.png) | ![image](https://user-images.githubusercontent.com/6129479/87164710-18ca9000-c297-11ea-9070-56c032ab2dc0.png)
Search Results | ![image](https://user-images.githubusercontent.com/6129479/87164598-f6d10d80-c296-11ea-81bf-eef1c588aeb4.png) | ![image](https://user-images.githubusercontent.com/6129479/87164682-0fd9be80-c297-11ea-9b56-bede2ec10573.png)
